src/test/java/MathematicsTest.java

```diff
@@ -0,0 +1,63 @@
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static org.testng.AssertJUnit.assertTrue;
+
+/**
+ * Created by LiudaBalatskaya on 5/8/2017.
+ */
+public class MathematicsTest {
+    public int result;
+    public Mathematics calculator;
+
+    @BeforeClass
+    public static void beforeClass(){
+        Mathematics.logger.info("In beforeClass method");
+    }
+    @BeforeMethod
+    public void initializerTest() {
+        calculator = new Mathematics();
+        calculator.setResult(0);
+    }
+    @AfterClass
+    public static void finishedClass(){
+        Mathematics.logger.info("Testing is finished");
+    }
+    @Test(groups = {"smoke", "fast"})
+    public void addCheck() {
+        calculator.add(2,2);
```

Comment:
create variables foe each value. Example: int firstValueForAdd = 2; etc.
Apply this rule in code below